app/proguard-config.gradle

```diff
@@ -18,10 +18,12 @@ ext {
     releaseProguardFiles = [
             './proguard-rules/android-support-rules.pro',
             './proguard-rules/autovalue-rules.pro',
-            './proguard-rules/okhttp2-rules.pro',
+            './proguard-rules/okhttp3-rules.pro',
             './proguard-rules/picasso-rules.pro',
-            './proguard-rules/retrofit-rules.pro',
+            './proguard-rules/retrofit2-rules.pro',
             './proguard-rules/retrolambda-rules.pro',
-            './proguard-rules/rxjava-rules.pro'
+            './proguard-rules/rxjava-rules.pro',
```

Comment:
We're using RxJava2 now, so the `rxjava-rules.pro` file should be renamed. Looking at the contents, it seems the rules are no longer relevant (certainly the packages have changed and the classes no longer exist). Given that RxJava2 will no doubt be quite volatile for the next months, I would be inclined to rename the file to `rxjava2-rules.pro` and  have the contents only with a comment:
```
# Currently no rules necessary
```